templates/terraform/monster-infrastructure/encode.tf

```diff
@@ -0,0 +1,16 @@
+module encode {
+  source = "/templates/processing-project"
+  providers = {
+    google.target = google.encode,
+    vault.target = vault.target
+  }
+
+  project_name = "broad-dsp-monster-encode-${local.env}"
+  is_production = var.is_production
+  command_center_argo_account_email = module.command_center.encode_argo_runner_email
+  region = "us-central1"
```

Comment:
Ah, this I would definitely change to one of the US-west regions. ENCODE is [hosted](https://registry.opendata.aws/encode-project/) in us-west-2 ([Oregon](https://docs.aws.amazon.com/general/latest/gr/rande.html)), so I think the closest GCP region would be us-west-1 (Also [Oregon](https://cloud.google.com/compute/docs/regions-zones/#locations))